tests/lava/lib/optimization/apps/schduler/test_solver.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+import pprint
+import unittest
+import os
+
+import numpy as np
+
+from lava.lib.optimization.apps.scheduler.problems import (
+    SchedulingProblem, SatelliteScheduleProblem)
+from lava.lib.optimization.apps.scheduler.solver import (Scheduler,
+                                                         SatelliteScheduler)
+
+
+def get_bool_env_setting(env_var: str):
+    """Get an environment variable and return True if the variable is set to
+    1 else return false.
+    """
+    env_test_setting = os.environ.get(env_var)
+    test_setting = False
+    if env_test_setting == "1":
```

Comment:
Can be shortened to
'return env_test_setting == "1"'

Reply:
Fair point.

Reply:
Just wanted to be consistent with other implementations of this function. This function is actually a code duplication. It's been implemented many times in the entire codebase.